pjlink/status.go

```diff
@@ -0,0 +1,91 @@
+package pjlink
+
+import "strings"
+
+type PowerStatus struct {
+	Power string `json:"power",omitempty`
+}
+
+type BlankedStatus struct {
+	Blanked bool `json:"blanked",omitempty`
+}
+
+type MuteStatus struct {
+	Mute bool `json:"muted",omitempty`
+}
+
+type VideoInput struct {
+	Input string `json:"input",omitempty`
+}
+
+type InputList struct {
+	Inputs []string `json:"inputs",omitempty`
+}
```

Comment:
I would pull all these out and put them in a common package (in the AV-API or something) so that ALL of the microservices will return the same thing.